packages/http-client-csharp/generator/Directory.Build.props

```diff
@@ -25,7 +25,7 @@
       Disable NU5105 NuGet Pack warning that the version is SemVer 2.0.
       SemVer 2.0 is supported by NuGet since 3.0.0 (July 2015) in some capacity, and fully since 3.5.0 (October 2016).
     -->
-    <NoWarn>$(NoWarn);NU5105</NoWarn>
+    <NoWarn>$(NoWarn);NU5105;SA1117;SA1101</NoWarn>
```

Comment:
This was needed because of the target framework upgrade. Should probably be handled in editorconfig instead.